03.asynchronous/callback_without_err.js

```diff
@@ -0,0 +1,19 @@
+import sqlite3 from "sqlite3";
+
+const db = new sqlite3.Database(":memory:");
+
+db.run(
+  "CREATE TABLE books (id INTEGER PRIMARY KEY AUTOINCREMENT, title TEXT NOT NULL UNIQUE);",
+  () => {
+    db.run('INSERT INTO books(title) VALUES ("よくわかるJavaScript")', () => {
```

Comment:
SQL のスタイルが一貫してなさそうです :eyes: `CREATE TABLE ...` と合わせるなら以下のようにする必要があると思いますー。それから SQL の文字列を表現するときに使うのはダブルクォートではなくシングルクォートです。あとこの SQL だけ末尾のセミコロンがないですー。

```suggestion
    db.run('INSERT INTO books (title) VALUES ("よくわかるJavaScript")', () => {
```